package.json

```diff
@@ -397,9 +397,21 @@
       "title": "Lint",
       "type": "object",
       "properties": {
+        "tool": {
+          "title": "Linter",
+          "description": "The linter that is used to lint your code",
+          "type": "string",
+          "default": "gometalinter",
+          "enum": [
+            "gometalinter",
+            "revive",
+            "golangci-lint"
+          ],
+          "order": 1
+        },
         "args": {
-          "title": "gometalinter Arguments",
-          "description": "Arguments to be passed when invoking `gometalinter`. Please ensure the `--json` argument is always included. Arguments are comma-separated.",
+          "title": "Linter Arguments",
+          "description": "Arguments to be passed when invoking the provided linter. Arguments are comma-separated.",
```

Comment:
For `additionalBuildArgs` and `additionalTestArgs` we use a config type of `string` and parse them with `yargs-parser`. 

We should probably be consistent for all config properties that represent arguments to feed into an external tool.

I prefer `string` over `array` as the string is more natural and looks exactly as you'd type it into a shell, rather than comma separating the args just to appease atom's config parser.

Reply:
Makes sense. The only thing is backward compatibility. I would have to use `args.join(' ')` in case the args are still an array in the config. Maybe even write them back to the config afterwards.